rackcity/utils/change_planner_utils.py

```diff
@@ -105,64 +101,74 @@ def detect_conflicts_cp(sender, **kwargs):
     ).update(asset_conflict_asset_number=asset)
 
 
+def get_change_plan(change_plan_id):
+    try:
+        change_plan = ChangePlan.objects.get(id=change_plan_id)
+    except ObjectDoesNotExist:
+        return (
+            None,
+            JsonResponse(
+                {
+                    "failure_message": Status.ERROR.value
+                    + "Change plan"
+                    + GenericFailure.DOES_NOT_EXIST.value,
+                    "errors": "Invalid change_plan Parameter",
+                },
+                status=HTTPStatus.BAD_REQUEST,
+            ),
+        )
+    else:
+        return change_plan, None
+
+
 def get_filtered_decommissioned_assets_for_cp(change_plan, data):
     decommissioned_assets = DecommissionedAsset.objects.all()
-    decomissioned_assets_cp = AssetCP.objects.filter(
-        change_plan=change_plan,
-        is_decommissioned=True,
+    decommissioned_assets_cp = AssetCP.objects.filter(
+        change_plan=change_plan, is_decommissioned=True,
     )
     decommissioned_assets, filter_failure_response = get_filtered_query(
-        decommissioned_assets,
-        data,
+        decommissioned_assets, data,
     )
     if filter_failure_response:
-        return (None, None, filter_failure_response)
-    decomissioned_assets_cp, filter_failure_response = get_filtered_query(
-        decomissioned_assets_cp,
-        data,
+        return None, None, filter_failure_response
+    decommissioned_assets_cp, filter_failure_response = get_filtered_query(
+        decommissioned_assets_cp, data,
     )
     if filter_failure_response:
-        return (None, None, filter_failure_response)
-    return (decommissioned_assets, decomissioned_assets_cp, None)
+        return None, None, filter_failure_response
+    return decommissioned_assets, decommissioned_assets_cp, None
```

Comment:
6. Renamed some variables, changing CP to _cp and decomissioned to decommissioned